slatedb/src/compactor.rs

```diff
@@ -668,12 +668,17 @@ impl CompactorEventHandler {
         self.state.db_state().log_db_runs();
         let compactions = self.state.compactions();
         for compaction in compactions {
-            info!("in-flight compaction [compaction={}]", compaction);
+            if log::log_enabled!(log::Level::Debug) {
```

Comment:
what's the intent here? why not just always log it at info?

Reply:
:? Is mix more verbose, which is useful for debugging. Didn't want to fill the logs with it at info level since it's pretty noisy.